package.json

```diff
@@ -1,57 +1,59 @@
 {
-  "name" : "@scm-manager/scm-repository-template-plugin",
-  "private" : true,
-  "version" : "1.0.0-SNAPSHOT",
-  "license" : "MIT",
-  "main" : "./src/main/js/index.tsx",
-  "scripts" : {
-    "build" : "ui-scripts plugin",
-    "watch" : "ui-scripts plugin-watch",
-    "test" : "jest",
-    "postinstall" : "ui-plugins postinstall"
+  "name": "@scm-manager/scm-repository-template-plugin",
+  "private": true,
+  "version": "1.0.0-SNAPSHOT",
+  "license": "MIT",
+  "main": "./src/main/js/index.tsx",
+  "scripts": {
+    "build": "ui-scripts plugin",
+    "watch": "ui-scripts plugin-watch",
+    "test": "jest",
+    "postinstall": "ui-plugins postinstall"
   },
-  "dependencies" : {
-    "@scm-manager/ui-plugins" : "2.0.0",
-    "@scm-manager/ui-components" : "^2.0.0-rc7",
-    "@scm-manager/ui-extensions" : "^2.0.0-rc7",
-    "classnames" : "^2.2.6",
-    "query-string" : "^5.0.1",
-    "react" : "^16.10.2",
-    "react-i18next" : "^10.13.1",
-    "react-redux" : "^5.0.7",
-    "react-router-dom" : "^5.1.2",
-    "redux" : "^4.0.0",
-    "styled-components" : "^4.4.0"
+  "dependencies": {
+    "@scm-manager/ui-plugins": "2.0.0",
```

Comment:
We should keep in mind that this should be 2.5.0 as sonn as the core is released.